apps/meteor/app/reactions/server/setReaction.ts

```diff
@@ -101,33 +86,62 @@ async function setReaction(room: IRoom, user: IUser, message: IMessage, reaction
 		await Messages.setReactions(message._id, message.reactions);
 		if (isTheLastMessage(room, message)) {
 			await Rooms.setReactionsInLastMessage(room._id, message.reactions);
-			void notifyOnRoomChangedById(room._id);
 		}
-		await callbacks.run('afterSetReaction', message, { user, reaction, shouldReact });
+
+		void callbacks.run('afterSetReaction', message, { user, reaction, shouldReact: true });
 
 		isReacted = true;
 	}
 
-	await Apps.self?.triggerEvent(AppEvents.IPostMessageReacted, message, user, reaction, isReacted);
+	void Apps.self?.triggerEvent(AppEvents.IPostMessageReacted, message, user, reaction, isReacted);
 
+	void notifyOnRoomChangedById(room._id);
```

Comment:
This was triggered only if it was the last message...

Reply:
oops